app/src/androidTest/java/org/schabi/newpipe/GroupedTests.java

```diff
@@ -0,0 +1,106 @@
+package org.schabi.newpipe;
```

Comment:
Why not in regular testfolder with the other tests?

Reply:
Never mind, i guess u need an emulator

Reply:
Yeah exactly. And I'm going to try to do the tests with Mockito but not sure how that will go. But these work with an emulator.